lstchain/scripts/lstchain_mc_r0_to_dl1.py

```diff
@@ -42,7 +43,11 @@
                     help='Path to a configuration file. If none is given, a standard configuration is applied',
                     default=None
                     )
-
+parser.add_argument('--dynamic-cleaning', action='store',
+                    type=lambda x: bool(strtobool(x)),
+                    dest='dynamic_cleaning',
```

Comment:
dest is not needed, since the same value is auto-generated from your `--dynamic-cleaning` argument.